src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/FutureWorkInterestsEntity.kt

```diff
@@ -0,0 +1,118 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import org.hibernate.annotations.UuidGenerator
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * Represents a Prisoner's future work aspirations, including the type/sector of work and their desired role within it.
+ */
+@Table(name = "future_work_interests")
+@Entity
+@EntityListeners(AuditingEntityListener::class)
+class FutureWorkInterestsEntity(
+  @Id
+  @GeneratedValue
+  @UuidGenerator
+  var id: UUID? = null,
+
+  @Column(updatable = false)
+  @field:NotNull
+  var reference: UUID? = null,
+
+  @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
+  @JoinColumn(name = "work_interests_id", nullable = false)
+  var interests: List<WorkInterestEntity>? = null,
+
+  @Column(updatable = false)
+  @CreationTimestamp
+  var createdAt: Instant? = null,
+
+  @Column(updatable = false)
+  @CreatedBy
+  var createdBy: String? = null,
+
+  @Column
+  @UpdateTimestamp
+  var updatedAt: Instant? = null,
+
+  @Column
+  @LastModifiedBy
+  var updatedBy: String? = null,
+)
+
+@Table(name = "work_interest")
+@Entity
+class WorkInterestEntity(
+  @Id
+  @GeneratedValue
+  @UuidGenerator
+  var id: UUID? = null,
+
+  @Column(updatable = false)
+  @field:NotNull
+  var reference: UUID? = null,
+
+  @Column
+  @Enumerated(value = EnumType.STRING)
+  @field:NotNull
+  var workType: WorkInterestType? = null,
+
+  @Column
+  var workTypeOther: String? = null,
+
+  @Column
+  var role: String? = null,
+
+  @Column(updatable = false)
+  @CreationTimestamp
+  var createdAt: Instant? = null,
+
+  @Column(updatable = false)
+  @CreatedBy
+  var createdBy: String? = null,
+
+  @Column
+  @UpdateTimestamp
+  var updatedAt: Instant? = null,
+
+  @Column
+  @LastModifiedBy
+  var updatedBy: String? = null,
```

Comment:
Every entity also has these audit type fields, but not the `created/updatedAtPrison` or `displayName` fields.

Reply:
Why do the other entities (and presumably domain classes) not have the prison and display name fields?
I have a funny feeling that we'll need them on some of the entities, because I think the screens show who last updated a given section of the CIAG Induction and in what prison.

I could be wrong, and I'm happy to go with it as is - we can always add them in later once we get into the detail of this stuff 👍  (Rome wasn't built in a day 😁 )

Reply:
The problem is that I have no idea what's required! Not now or in the future. I originally added all those fields to each entity, but it seemed overkill.